code/admin/SubsiteAdmin.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace SilverStripe\Subsites\Admin;
```

Comment:
Yeah, if you move code to src then you can also capitalise the subfolders at the same time. Otherwise you'll need to do something like `git mv admin blah && git mv blah Admin` since it won't pick up the case change otherwise (at least on OSX anyway)